.env.example

```
@@ -24,6 +24,9 @@ QDRANT_COLLECTION=memories_arctic1024
 # dev-open flag). GRAPH_API_KEY guards the FalkorDB bridge.
 GRAPH_API_KEY=
 ALAYA_API_KEY=
+# Optional second bearer for headless read-only consumers (pure reads only,
+# 403 on every mutating route incl. /store). Must differ from ALAYA_API_KEY.
+ALAYA_READONLY_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALAYA_READONLY_API_KEY key should go before the GRAPH_API_KEY key

(UnorderedKey)